west.yml

```diff
@@ -124,7 +124,7 @@ manifest:
       groups:
         - hal
     - name: hal_ti
-      revision: 1992a4c536554c4f409c36896eda6abdc414d277
+      revision: pull/28/head
```

Comment:
We will need to hold off on merging this until zephyrproject-rtos/hal_ti#28 is merged, and then this line will need to be updated with the appropriate revision.

Reply:
I would guess that this also will need zephyrproject-rtos/hal_ti#27

Reply:
Yes, you are right, I think it requires the updated HAL TI because of a `CMD_PROP_RADIO_SETUP_PA` addition (although it's an alias to the old radio setup command, I haven't tested the PA without the SDK update):
```c
#define CMD_PROP_RADIO_SETUP_PA                                 CMD_PROP_RADIO_SETUP
```